pages/services/elastic/2.5.0-6.3.2/how-to-guides/index.md

```diff
@@ -0,0 +1,262 @@
+---
```

Comment:
Just curious why only this page has text line-wrapped vs. the other docs.

Reply:
It's just my personal preference! Since I created the file I thought I could sneak some line-wrapped content (as god intended).

It probably makes sense to make it not line-wrapped to be consistent with the rest of the docs. I'll do that.

Reply:
https://github.com/mesosphere/dcos-docs-site/pull/2054/commits/d4cab96c71f7b26ce8b6225d6ffebdfea5cdaf81